src/main/java/org/schabi/newpipe/extractor/Subtitles.java

```diff
@@ -0,0 +1,31 @@
+package org.schabi.newpipe.extractor;
+
+public class Subtitles {
```

Comment:
Please add a datatype field to this class.
So we can distinguish between different formats.

Reply:
Yeah, and IMHO we should only have the language code and then get the name of the language in NewPipe (like we also do with the YouTube content language).